packages/block-library/src/navigation/edit/leaf-more-menu.js

```diff
@@ -24,7 +24,13 @@ const BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU = [
 	'core/navigation-submenu',
 ];
 
-function AddSubmenuItem( { block, onClose, expandedState, expand } ) {
+function AddSubmenuItem( {
```

Comment:
A copy of this code is also in the Navigation section of Browse Mode. Since we don't show the Link UI there I don't think we need to make these changes there, I'm just noting it in case I've missed anything.

Reply:
Good spot but it's going to be removed in https://github.com/WordPress/gutenberg/pull/50788 so no need to worry.